youtube_dl/extractor/cloudflarestream.py

```diff
@@ -28,17 +28,26 @@ class CloudflareStreamIE(InfoExtractor):
     }, {
         'url': 'https://watch.cloudflarestream.com/9df17203414fd1db3e3ed74abbe936c1',
         'only_matching': True,
+    }, {
+        'url': 'https://watch.videodelivery.net/9df17203414fd1db3e3ed74abbe936c1',
+        'only_matching': True,
+    }, {
+        'url': 'https://videodelivery.net/9df17203414fd1db3e3ed74abbe936c1',
```

Comment:
No such URL exists.

Reply:
True.  I don't have a working watch URL to add, should I just remove these?

Reply:
Yes.

Reply:
Ok.